frontend/frontend/src/components/GameSummary.vue

```diff
@@ -62,7 +73,16 @@ export default {
 	},
 	methods: {
 		routeToHome: function () {
-			this.$router.push("/home");
+			this.$router.push('/home');
+			//console.log("hey");
+		},
+		artistsList: function () {
+			var artistNames = [];
+			var cardSet = this.$route.params.cardSet;
+			cardSet.forEach((card) => {
+				artistNames.push(card.artistName);
+			});
+			return (artistNames)
```

Comment:
Let's move this to be a computed function, that way we are using vue to our advantage.
computed value code:

artistsList: function () {
			var artistNames = [];
			var cardSet = this.$route.params.cardSet;
			cardSet.forEach((card) => {
				if (artistNames.includes(card.artistName)) {
					return;
				}
				artistNames.push(card.artistName);
			});
			return artistNames;
		},

Reply:
I reccomend adding a check similar to what I have above, because currently you are showing 2 of each artist.

Reply:
thanks, all done